ui/packages/consul-ui/app/components/auth-profile/index.scss

```diff
@@ -0,0 +1,19 @@
+.auth-profile {
+  padding: 0.9em 1em;
+}
+.auth-profile {
+  @extend %p2;
+}
+.auth-profile dt span {
+  font-weight: var(--typo-weight-normal);
+}
+.auth-profile dt {
+  font-weight: var(--typo-weight-bold);
+}
+.auth-profile dt,
+.auth-profile dd {
+  color: rgb(var(--tone-gray-800));
+}
+.auth-profile dt span {
+  color: rgb(var(--tone-gray-600));
+}
```

Comment:
All of these styles were in `%menu-panel` for any `dt`s within a menu-panel. Pretty sure that the only instance of a dt in a menu-panel is this auth-profile/AuthProfile component (especially judging from the what the styles are doing)

So now that AuthProfile is a fully fledged component I picked out its styles that were slighly dotted around elsewhere and moved them all into the stylesheet specifically for this component.

Just thinking out loud for the future, this one would be a good one to switch to shadow DOM, why?:

1. There is no need for any `id` references so the id boundary/isolation of shadow DOM doesn't effect us
2. The styles are super simple
3. There is one instance of `@extend` which we would like to move away from and use shadow DOM features instead. As there is only a simple usage of `@extend` here it would be good to get a feel for what the replacement would look like in this instance.